docs/guides/es2k-guide.md

```diff
@@ -129,10 +129,8 @@ After installing ATE Kernel on HOST machine, install the following drivers
 to bind the network devices (netdevs) to the E2100 target.
 
 ```bash
+modprobe auxiliary 
 modprobe idpf
-modprobe auxiliary
-modprobe iecm
```

Comment:
Confirming that we do not need iecm and iavf?

Reply:
yes, iecm and iavf are merged as part of idpf